project/src/main/webapp/events-feed.html

```diff
@@ -5,6 +5,7 @@
     <title>STEP 188 Project</title>
     <link rel="stylesheet" href="events-feed.css" />
     <link rel="stylesheet" href="styles.css" />
+    <script src="https://kit.fontawesome.com/0d580893be.js" crossorigin="anonymous"></script>
```

Comment:
nit: Im wondering if there is a similar google font and you can use that js file.

Reply:
This isn't for fonts, it's for the icons